cli/internal/terraform/terraform/gcp/iam/main.tf

```
@@ -0,0 +1,69 @@
+terraform {
+  required_providers {
+    google = {
+      source  = "hashicorp/google"
+      version = "~> 4.43.0"
```

```suggestion
      version = "4.43.0"
```
We try to always pin versions.